apps/frontend/src/pages/onboarding/styles.module.css

```diff
@@ -62,9 +74,10 @@
 		font-size: 28px;
 		line-height: 30px;
 	}
+}
 
-	.button-container {
-		flex-direction: column;
-		max-width: 250px;
+@media (width <= 640px) {
```

Comment:
Currently we have only one breakpoint and it is 768px.

Reply:
I think we can also use other breakpoints as per [https://bsa2024.slack.com/archives/C07HBKZFCF2/p1724643817020249](url)